Error: Cannot find module 'pug' when using nodejs-express production image

**Describe the bug**
I have an application developed using the nodejs-express stack.  It works fine when running it using "appsody run".  However when I use "appsody deploy" the container fails with this error message:

Error: Cannot find module 'pug'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at Module.newFunc (/project/node_modules/appmetrics/lib/aspect.js:101:26)
    at Module.newFunc [as require] (/project/node_modules/ibmapm-embed/appmetrics-zipkin/lib/aspect.js:101:26)
    at require (internal/modules/cjs/helpers.js:25:18)
    at new View (/project/node_modules/express/lib/view.js:81:14)
    at Function.render (/project/node_modules/express/lib/application.js:570:12)
    at ServerResponse.render (/project/node_modules/express/lib/response.js:1008:7)
    at /project/user-app/quote.js:14:7

**To Reproduce**
Run an application that requires a view engine to render pages.

**Expected behavior**
Application works like it did locally.

**Actual behaviour**
See above.

**Environment Details (please complete the following information):**
 - OS: Windows 10
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

**If applicable please specify:**
 - CLI version: 0.4.2
 - Stack you are using: nodejs-express

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
I compared the development Dockerfile image (Dockerfile-stack) and the production image and noticed that the former sets the NODE_PATH environment variable to the node_modules folder in the user's app while the latter does not.

ENV NODE_PATH=/project/user-app/node_modules

I updated my deployment yaml to set this environment variable and the container now worked as expected.
```
        env:
          - name: NODE_PATH
            value: /project/user-app/node_modules
```

I assume the production Dockerfile should be setting this variable.

--- a/incubator/nodejs-express/image/config/app-deploy.yaml
+++ b/incubator/nodejs-express/image/config/app-deploy.yaml
@@ -19,6 +19,7 @@ spec:
       port: APPSODY_PORT
     initialDelaySeconds: 5
     periodSeconds: 2
+    timeoutSeconds: 1
   livenessProbe:
     failureThreshold: 12
     httpGet:

--- a/incubator/nodejs-express/image/project/Dockerfile
+++ b/incubator/nodejs-express/image/project/Dockerfile
@@ -36,6 +36,8 @@ COPY --chown=node:node --from=0 /project/user-app/node_modules /project/user-app
 
 WORKDIR /project
 
+ENV NODE_PATH=/project/user-app/node_modules
+
 ENV NODE_ENV production
 ENV PORT 3000
 

--- a/incubator/nodejs-express/image/project/package.json
+++ b/incubator/nodejs-express/image/project/package.json
@@ -1,6 +1,6 @@
 {
   "name": "nodejs-express",
-  "version": "0.2.5",
+  "version": "0.2.6",
   "description": "Node.js Express Stack",
   "license": "Apache-2.0",
   "repository": {

--- a/incubator/nodejs-express/stack.yaml
+++ b/incubator/nodejs-express/stack.yaml
@@ -1,5 +1,5 @@
 name: Node.js Express
-version: 0.2.5
+version: 0.2.6
 description: Express web framework for Node.js
 license: Apache-2.0
 language: nodejs

